config/data_sync_list/ibm.json

```diff
@@ -0,0 +1,10 @@
+{
+    "Files": [
+        {
+            "Path": "/var/lib/ibm/bmcweb/RootCert",
+            "Description": "Root certificate",
+            "SyncDirection": "Active2Passive",
```

Comment:
can bmcweb handle this being synced?

Reply:
Not just an example. Added the details as per the data we collected internally from every team.

Reply:
Was wondering what bmcweb does when it has this file changed without it knowing.  I guess it will be a while before it becomes an issue so I won't worry about it now.

Reply:
Yeah I got you. For now we can proceed with this approach. Because my plan was just to make a basic infrastructure for data-sync now. Later we will add the details of all files and directories, and at that time will  ask the respective component owners to review the same again.